Gems/EMotionFX/Code/MCore/Source/AlignedArray.h

```diff
@@ -18,7 +18,7 @@ namespace MCore
     /**
      * Dynamic array template, using aligned memory allocations.
      * This array template allows dynamic sizing. It also stores the memory category of the data.
-     * It can theoretically store 4294967296 items (maximum uint32 value).
+     * It can theoretically store 18446744073709551614 items (maximum size_t value - 1 for the invalid index).
      */
     template <typename T, uint32 alignment>
     class AlignedArray
```

Comment:
We can also get rid of the aligned array directly, or do it in a second pass.  The AZStd::vector<> is using aligned memory allocations already, so there is no use for this class anymore, too.

Reply:
I say we get this in first, then do a separate round for `AlignedArray`.